galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/TestClassesRas.java

```diff
@@ -65,7 +67,7 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
 		
 		/* looking for sort options in query and sorting accordingly */
 		try {
-			if(!ExtractQuerySort.isAscending(queryParams, "testclass")) {
+			if(!SortQueryParameterChecker.isAscending(queryParams, "testclass")) {
```

Comment:
use the variable, not the class name. If the variable was called something shorter it might help distinguish between the two.